src/relay/op/annotation/annotation.cc

```diff
@@ -201,5 +201,7 @@ TVM_REGISTER_GLOBAL("relay.op.annotation._make.compiler_end")
       return Call(op, {expr}, Attrs(attrs), {});
     });
 
+      TVM_DLL extern  const Op stop_fusion_op = Op::Get("annotation.stop_fusion");
+
```

Comment:
remove this?

Reply:
This is for MinGW register order , the same as vm.invoke_tvm_op